### ISSUE TITLE
Missing qemu-img in latest openqa_worker Docker images

### ISSUE THREAD

**Issue by ssssam** [os-autoinst/openQA#4102]
Steps to reproduce:

  * pull latest openqa_worker image: `podman pull registry.opensuse.org/devel/openqa/containers15.2/openqa_worker:latest` -> image ID 9d396a92c176
  * enter container and try to run qemu-img:
          podman run --privileged --rm -i -t  --entrypoint /bin/sh 9d396a92c1769902c227aeaa8ec64d9c423d86354c7c9e9219218c47bdca2c40 qemu-img
          sh-4.4# qemu-img
          sh: qemu-img: command not found
          sh-4.4# kvm-img
          sh: kvm-img: command not found

This causes a failure in the worker as seen here: https://openqa.gnome.org/tests/1

```
[2021-07-29T11:17:45.912 UTC] [info] ::: backend::baseclass::die_handler: Backend process died, backend errors are reported below in the following lines:
  no kvm-img/qemu-img found
[2021-07-29T11:17:45.912 UTC] [info] ::: OpenQA::Qemu::Proc::save_state: Refusing to save an empty state file to avoid overwriting a useful one
```

Worked around by building an image ourselves from this repo commit ceabc2be82e1f1c6c3c51bf017c1b79494ace869, which does contain `qemu-img` and successfuly runs tests.

**Comment by okurz** [comment]
os-autoinst, the backend of openQA worker, supports multiple backends, not only qemu. So it's technically correct that the openQA worker does not provide qemu-img as only qemu jobs rely on that.

When you say "latest", do you see this as a regression and it worked differently in before?

**Comment by ssssam** [comment]
It works differently with an image that I built myself from https://github.com/os-autoinst/openQA/tree/master/container/worker. And I am told that older images didn't have this problem.

I would like an image that can be used 'as-is' with the qemu backend. If the images linked at https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#container-based-setup need extra steps before they can be used, we could improve how that's documented.

**Comment by okurz** [comment]
Of course, I agree. With this your reported issue is certainly valid. Our container based setup has so far some "rough edges" still :)

**Comment by okurz** [comment]
> Note that in the meantime #4196 and #4196 have been merged.

you mentioned the same PR twice. Was there a second one?
